### ISSUE TITLE
Interactions with select menus don't seem to be received

### ISSUE THREAD

**Issue by lolgeny** [serenity-rs/serenity#1428]
For example, having
```rust
                    channel.send_message(ctx, |m|
                        m.content("** **").components(|c|
                            c.create_action_row(|r|
                                r.create_select_menu(|m|
                                    m.custom_id("platform")
                                        .options(|o|
                                            o.create_option(|o|
                                                o.label("Java").default_selection(true).value("java")
                                            )
                                            .create_option(|o|
                                                o.label("Bedrock").value("bedrock")
                                            )
                                        )
                                )
                            )
                            .create_action_row(|r|
                                r.create_select_menu(|m|
                                    m.custom_id("version")
                                    .options(|o|
                                        o.create_option(|o|
                                            o.label("1.17.1").default_selection(true).value("v_1.17.1")
                                        )
                                        .create_option(|o|
                                            o.label("21w99a").value("v_21w99a")
                                        )
                                    )
                                )
                            )
                            .create_action_row(|r|
                                r.create_select_menu(|m|
                                    m.custom_id("topics")
                                    .options(|o|
                                        o.create_option(|o|
                                            o.label("Loot Tables").value("t_loot_tables")
                                        )
                                        .create_option(|o|
                                            o.label("Shaders").value("t_shaders")
                                        )
                                        .create_option(|o|
                                            o.label("Text Components").value("t_text_components")
                                        )
                                    )
                                    .min_values(0)
                                    .max_values(3)
                                )
                            )
                            .create_action_row(|r|
                                r.create_button(|b|
                                    b.label("Create").style(ButtonStyle::Primary).custom_id("create_thread")
                                )
                            )
                        )
                    ).await.unwrap();
```

Does not respond to either

```rust
message.await_component_interaction(&ctx).await
```

or

```rust
EventHandler::interaction_create
```

However, in the same project, removing the rows with the select menus and just having the button seems to receive interactions just fine

**Comment by tazz4843** [comment]
Can reproduce with serenity as a Git dependency (branch `current`). The more interesting part is that this used to work fine just yesterday.

Source for the dropdowns is here: https://github.com/tazz4843/scripty/blob/main/src/cmd_setup.rs#L113L144

I can also reproduce that buttons alone work perfectly fine.

**Comment by HarmoGlace** [comment]
Thank you for these informations, can the sent message (with the select menu) be successfuly deserialized?

**Comment by Zaitara** [comment]
A fix was made and is merged, see #1418. Reopen if you can reproduce on latest commit.

**Comment by lolgeny** [comment]
Seems to work on the latest commit, thanks
